src/modules/launcher/Plugins/Microsoft.Plugin.Folder/ContextMenuLoader.cs

```diff
@@ -39,7 +39,7 @@ public List<ContextMenuResult> LoadContextMenus(Result selectedResult)
                 contextMenus.Add(new ContextMenuResult
                 {
                     PluginName = Assembly.GetExecutingAssembly().GetName().Name,
-                    Title = _context.API.GetTranslation("Microsoft_plugin_folder_copy_path"),
+                    Title = Properties.Resources.Microsoft_plugin_folder_copy_path,
```

Comment:
Do we still need `GetTranslation` to be part of `IPublicAPI` interface?

Reply:
No, it's used only for Wox's implementation of localization using xaml files. For resx files it is sufficient to directly access the resource. GetTranslation also makes the code more complex since it loads the resources from all the plugins into a single class object, causing lesser abstraction between projects and higher chance of bugs getting introduced from typos in resource names, since it does not throw an error if a resource is missing. From the source code all the references of the `GetTranslation` method are only for accessing resources like this.